proco/connection_statistics/models.py

```diff
@@ -45,6 +46,9 @@ class CountryWeeklyStatus(TimeStampedModel, models.Model):
     connectivity_speed = models.FloatField(blank=True, default=0.0)
     integration_status = models.PositiveSmallIntegerField(choices=INTEGRATION_STATUS_TYPES, default=JOINED)
     avg_distance_school = models.FloatField(blank=True, default=0.0)
+    schools_with_data_percentage = models.DecimalField(
+        decimal_places=2, max_digits=6, default=0, validators=[MaxValueValidator(100), MinValueValidator(0)],
```

Comment:
do we have reasons to validate automatically updated readonly fields?

Reply:
Is it bad to have additional validation?

Reply:
nope. just interesting why now :)